internal/locate/region_request.go

```diff
@@ -2360,3 +2360,13 @@ func (s *replicaSelector) patchRequestSource(req *tikvrpc.Request, rpcCtx *RPCCo
 	}
 	sb.WriteString(req.ReadType)
 }
+
+func isReadReq(tp tikvrpc.CmdType) bool {
```

Comment:
How about the read requests triggered by write statements?

Reply:
Nice catch, already refined the logic, PTAL.